CHANGELOG.md

```diff
@@ -8,6 +8,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ## Unreleased
 
 <!-- List New Changes Here -->
+* Build now delegates authority to the Analyzer for what urls are package external instead of local heuristics.
+* Bundling now processes files coming in from streams, to support things like js-minification before bundling.
 
```

Comment:
Does this PR also mean that we'll be able to print warnings with source ranges in `polymer build`? Is there a bug to track this?

It'll be a huge win once we can. Just this morning someone reported an error in their build that – I think – would have been obvious with a source range but was pretty opaque without.